x-pack/plugin/esql/compute/src/test/java/org/elasticsearch/compute/aggregation/blockhash/CategorizeBlockHashTests.java

```diff
@@ -241,14 +264,14 @@ public void testCategorize_withDriver() {
                 BytesRefVector.Builder textsBuilder = driverContext.blockFactory().newBytesRefVectorBuilder(10);
                 LongVector.Builder countsBuilder = driverContext.blockFactory().newLongVectorBuilder(10)
             ) {
-                textsBuilder.appendBytesRef(new BytesRef("a"));
-                textsBuilder.appendBytesRef(new BytesRef("b"));
+                textsBuilder.appendBytesRef(new BytesRef("aaazz"));
```

Comment:
The categorization analyzer skips numbers, including hex numbers. Hence these changes.

Reply:
Not sure where to exactly add this as a comment. I'll at least document the skipping "DB" in the csv test

Reply:
I'll add a note here too.